pkg/noderesourcetopology/filter_test.go

```diff
@@ -231,6 +231,66 @@ func TestNodeResourceTopology(t *testing.T) {
 			node:       nodes[0],
 			wantStatus: nil,
 		},
+		{
+			name: "Best effort QoS requesting devices, Pod Scope Topology policy; pod fit",
+			pod: makePodByResourceList(&v1.ResourceList{
+				nicResourceName: *resource.NewQuantity(5, resource.DecimalSI)}),
+			node:       nodes[2],
+			wantStatus: nil,
+		},
+		{
+			name: "Best effort QoS requesting CPU (enough on NUMA) and devices, Pod Scope Topology policy; pod fit",
```

Comment:
let's make sure to fix the name of the testcases to be consistent with the expected pod QoS

Reply:
Fixed. Thanks